Loop/Managers/DeviceAlert/DeviceAlertManager.swift

```diff
@@ -19,24 +20,38 @@ protocol DeviceAlertManagerResponder: class {
 /// - serializing alerts to storage
 /// - etc.
 public final class DeviceAlertManager {
+    static let soundsDirectory = "\(NSSearchPathForDirectoriesInDomains(FileManager.SearchPathDirectory.libraryDirectory, .userDomainMask, true).last!)/Sounds"
```

Comment:
[nit] You might consider using the FileManager API to grab this URL: https://developer.apple.com/documentation/foundation/filemanager/1407726-urls